### PR TITLE
Azure stack: Add samples under Hybrid

### PR DESCRIPTION
**Pull request by viananth**

### CODE REVIEW

File: Hybrid/example.rb [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      ENV['AZURE_CLIENT_SECRET'],
+      active_directory_settings
+      )
+  credentials = MsRest::TokenCredentials.new(provider)
+
+  options = {
+      credentials: credentials,
+      subscription_id: subscription_id,
+      active_directory_settings: active_directory_settings,
+      base_url: 'https://management.<location>.azurestack.external'
```
Comment (bganapa) [inline Hybrid/example.rb]: Example should be runnable as such against ASDK one node deployment. I think we should hardcode the location to be local in the example.rb

Comment (bganapa) [reply Hybrid/example.rb]: or base url could be a environment var just like tenant id

Comment (bganapa) [reply Hybrid/example.rb]: this comment is applicable for other samples as well

File: Hybrid/spec/example_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
```
Comment (bganapa) [inline Hybrid/spec/example_spec.rb]: Whats this file for?

File: Hybrid/example.rb [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    if props.respond_to? str.to_sym
+      puts "\t\t#{str}: #{props.send(str.to_sym)}"
+    end
+  end
+  puts "\n\n"
+end
+
+def get_active_directory_settings()
+  settings = MsRestAzure::ActiveDirectoryServiceSettings.new
+  settings.authentication_endpoint = 'https://login.windows.net/'
```
Comment (bganapa) [inline Hybrid/example.rb]: These should be read from ARM metadata endpoint.  On ADFS environment authentication_endpoint is diferent

**Review by bganapa — CHANGES_REQUESTED** [review]

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**